docs/source/caching/http-cache.mdx

```diff
@@ -89,4 +89,12 @@ val response = apolloClient.query(query)
 
 ## Clearing the cache
 
-Call `apolloClient.httpCache.clearAll()` to clear the cache of all entries.
+Call below to clear the cache of all entries, note that retrieving `apolloClient.apolloStore` will throw an exception if a cache has not been configured on the client.
+
+```kotlin
+try {
+    apolloClient.apolloStore.clearAll()
```

Comment:
I think `apolloClient.httpCache.clearAll()` was correct in the context of the http cache. `apolloClient.apolloStore.clearAll()` is for the Normalized Cache.

Reply:
@BoD I see, thanks, should I delete this section then and move it to the caching page? Since `apolloClient.httpCache` is no longer available in version 4.

Reply:
`httpCache` is still [there](https://github.com/apollographql/apollo-kotlin/blob/main/libraries/apollo-http-cache/src/main/kotlin/com/apollographql/apollo3/cache/http/HttpCacheExtensions.kt#L182) in v4. The http cache is a dedicated dependency that has to be added to the project though, so maybe that's why you don't see it?

Reply:
🤦 
[revert http cache doc changes, add clear cache section to normalized doc](https://github.com/apollographql/apollo-kotlin/pull/5289/commits/37df7c09b99c35d5c6fe25962a0342fe9dcda752)